packages/web/src/common/store/queue/sagas.ts

```diff
@@ -409,13 +425,22 @@ export function* watchPrevious() {
       // deleted or to which the user does not have access, skip over it.
       if (
         track &&
-        (track.is_delete || user?.is_deactivated || !doesUserHaveAccess)
+        (track.is_delete ||
+          user?.is_deactivated ||
+          (!doesUserHaveAccess && !track.preview_cid))
```

Comment:
why would we want to skip over a preview if going backwards?

Reply:
oh nvm i get it

Reply:
i added this bc if the track is "not previewable" then we want to skip it. tho i guess this isn't strictly necessary bc we shouldn't be able to get to such a case?